src/main/java/org/rumbledb/compiler/TranslationVisitor.java

```diff
@@ -636,13 +640,13 @@ public Void visitMultiplicativeExpr(JsoniqParser.MultiplicativeExprContext ctx)
     @Override
     public Void visitInstanceOfExpr(JsoniqParser.InstanceOfExprContext ctx) {
         Expression mainExpression;
-        FlworVarSequenceType sequenceType;
+        SequenceType sequenceType;
```

Comment:
Can we apply the idea we have applied to TreatExpr to all applicable places in this file? These variables do not need to be defined in this outermost context. They can simply be defined within the if statement where they are used.

Reply:
DONE